src/components/Code.tsx

```diff
@@ -195,7 +195,15 @@ const lightStyles = StyleSheet.create({
   'hljs-attr': {color: lightColors.text}, // Attributes within JSX Tags
 });
 
-function renderLowLightNode(node: lowlight.AST.Unist.Node) {
+function createTextKey(text: string, index: number) {
```

Comment:
Include some comment here about why you're using this as your key. Maybe with some example string so it's clear what the source data looks like.